infra/modules/gcp-secrets/main.tf

```diff
@@ -5,6 +5,9 @@
 
 locals {
   replica_locations = coalesce(var.replica_regions, var.replica_locations)
+
+  terraform_managed_secrets  = { for k, v in var.secrets : k => v if v.value_managed_by_tf }
```

Comment:
`terraform_managed_secrets` as var name makes more sense in AWS, but here as we are going to create or not the versions, I'd rename it `terraform_managed_secrets_versions` or something like that

Reply:
Yeah, as https://github.com/Worklytics/psoxy/pull/419/files#diff-169b5a139f9d65376c18eb220ce34546a3fde51283399f09e28cc60c7ce224e9L11 it could be confusing why we are creating all secrets coming from `var.secrets` but only versions from `terraform_managed_secrets`. I think that the rename worth it...

Reply:
ok, yeah - tried to clean it up.